src/Components/Components/src/Microsoft.AspNetCore.Components.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    $CommonMetadataElements$
+    <dependencies>
+      <group targetFramework=".NETStandard2.0">
+        <dependency id="Microsoft.AspNetCore.Components.Analyzers" version="$coherentPackageVersion$" />
+        <dependency id="Microsoft.AspNetCore.Authorization" version="$coherentPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.JSInterop" version="$jsInteropPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="System.ComponentModel.Annotations" version="$systemComponentModelAnnotationsPackageVersion$" exclude="Build,Analyzers" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="$outputPath$$assemblyName$.dll" target="lib\netstandard2.0\$assemblyname$.dll" />
```

Comment:
snupkg? Not entirely sure whatchu mean.

Reply:
With KoreBuild, we used to have to do some funky things to get it build the symbols nupkg correctly \ at all:

https://github.com/aspnet/AspNetCore/blob/release/2.2/src/Mvc/Mvc.Razor/src/Microsoft.AspNetCore.Mvc.Razor.csproj#L27-L37.

Do we need that here?

Reply:
1. `$(IncludeSymbols)` is now `true` by default
2. `@(_DependencyPackageInfo->WithMetadataValue('PackageId', 'Microsoft.CodeAnalysis.Razor')->Metadata('PackageVersion'))` is the incantation I was looking for [below](https://github.com/aspnet/AspNetCore/pull/12027#discussion_r302363588). Compared to the old stuff, update the wrapper to use a `@(NuspecProperty)` item and I suspect `@(_DependencyPackageInfo)` won't work and you'll need to find the correct item group.

Reply:
Don't think so unless it generates symbols differently:

![image](https://user-images.githubusercontent.com/2008729/61069937-49e15d80-a3c2-11e9-8122-4c855d86c1a2.png)

Reply:
Symbols are included in *.symbols.nupkg files, not the main package. And, I don't see anything between this project and https://github.com/aspnet/AspNetCore/blob/master/Directory.Build.props#L44 to prevent generation of Microsoft.AspNetCore.Components.symbols.nupkg. If that's not happening, create a bin log and look deeper.

Reply:
Note for myself: Need to include the pdb file

Reply:
@dougbu I can't utilize `Reference` because it's not actually referenced (we don't want it to be). Any other suggestions?

Reply:
Hmm, actually I could probably utilize a project reference where I state to not reference the assembly.

Reply:
Yeah, something like
``` xml
    <Reference Include="Microsoft.AspNetCore.Components.Analyzers">
      <ExcludeAssets>all</ExcludeAssets>
      <GlobalPropertiesToRemove>TargetFramework</GlobalPropertiesToRemove>
      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
      <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
    </Reference>
```
(Not positive whether `%(ExcludeAssets)` helps but it won't hurt.)

Reply:
Can't use `<Reference>` because the analyzer isn't in the shared framework. Also doesn't look like I can use `ProjectReference` ReferenceOutputAssembly` `false` because it doesn't provide enough information as to what package version it generates. Are we sure utilizing `$(PackageVersion)` isn't sufficient? These are projects in the same solution built as part of the same build graph.